DH_Engine/Classes/DHVoiceChatRoom.uc

```diff
@@ -54,6 +95,29 @@ simulated event bool IsMember(PlayerReplicationInfo PRI, optional bool bNoCascad
     return bool(GetMask() & (1 << PRI.VoiceID));
 }
 
+simulated function array<PlayerReplicationInfo> GetMembers()
+{
+    local array<PlayerReplicationInfo>      PRIArray;
+    local int                               i;
+
+    if (GRI != none && ValidMask())
```

Comment:
Yea we might just want to remove that function call, but for now lets leave it unless we find a bug at which point we will try removing it.  TMK that is a native function.